csrc/scheduler/matmul.cpp

```diff
@@ -1215,9 +1221,8 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
   // handle epilogue and always vectorize Ki
   if (params.use_smem_epilogue) {
     smem_epilogue->setMemoryType(MemoryType::Shared);
-    auto swizzled_dom = swizzleSharedMemory(smem_epilogue);
-    smem_epilogue->setLoopDomain(swizzled_dom.as<IterDomain*>());
-    smem_epilogue->setHasSwizzleOp();
+    auto swizzled_dom = swizzleSharedMemory<false>(smem_epilogue);
+    smem_epilogue->setAllocationDomain(swizzled_dom.as<IterDomain*>(), true);
```

Comment:
We no longer need to call `setHasSwizzleOp`?

Reply:
No, this is for legacy swizzle only. The indexing of legacy swizzle rely on this flag to know if a separate indexing pass is required to patch the index. But for the new swizzle, the index is just naturally correct in the first pass, there is nothing to patch.